website/src/corso-announcement-free-backup-for-microsoft-365.html

```diff
@@ -52,13 +52,31 @@ <h3 class="mb-3 text-3xl leading-normal font-medium text-white">A home to call y
+                                <h2 class="mb-4 text-3xl leading-normal font-medium" id="data-loss-happens">Data Loss Happens</h2>
 
-                            <div class="mt-6">
-                                <p class="text-slate-400">The most well-known dummy text is the 'Lorem Ipsum', which is said to have originated in the 16th century. Lorem Ipsum is composed in a pseudo-Latin language which more or less corresponds to 'proper' Latin. It contains a series of real Latin words. This ancient dummy text is also incomprehensible, but it imitates the rhythm of most European languages in Latin script.</p>
-                                <p class="text-slate-400 italic border-x-4 border-indigo-600 rounded-tl-xl rounded-br-xl mt-3 p-3">" There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. "</p>
-                                <p class="text-slate-400 mt-3">The advantage of its Latin origin and the relative meaninglessness of Lorum Ipsum is that the text does not attract attention to itself or distract the viewer's attention from the layout.</p>
-                            </div>
+                                <p>Let’s talk about the stats on data loss: </p>
+                                <p>Data loss can result from accidental or intentional deletion, cyber-attacks and malware, a poorly executed migration, or the cancellation of a software license, among other reasons. For example, <a href="https://www.veeam.com/blog/data-loss-2022.html">40% – 2 out of 5 servers</a> – had at least one or more outages over the past 12 months. And cybercrime is on a continual rise – the average number of data breaches and cyberattacks were <a href="https://www.forbes.com/sites/chuckbrooks/2022/06/03/alarming-cyber-statistics-for-mid-year-2022-that-you-need-to-know/?sh=642204357864">up 15.1% in 2021</a>, compared with the previous year. As of 2022, the average cost of a data breach in the U.S. was $9.44 million. </p>
+                                <h2 class="mb-4 text-3xl leading-normal font-medium" id="the-limitations-of-backup-tools">The limitations of backup tools</h2>
+                                <p class="text-base pb-4">While many backup and recovery solutions exist for M365 data, many lack comprehensive workflows, are hard to use and implement, or are cost prohibitive. Yet reliable and regular backups are critical for ensuring your data is protected and always available, in case of a breach, unexpected server failure or failed data migration. </p>
+                                <p><img title="image of tweet" alt="Tweet from Elon Musk: Part of today will be turning off the &quot;microservices&quot; bloatware. Less than 20% are actually needed for Twitter to work!" src="assets/images/blog/ian_tweet.png" class=" shadow">
```

Comment:
I see how you are trying to make this more topical but I see this as a stretch.

Reply:
revised